src/docs/docs.md

```diff
@@ -40,30 +40,47 @@ is derived from the docstrings of the classes implementing those symbols.
 Generating the site
 -------------------
 
+The site is generated with a script at `build-support/bin/publish_docs.sh`. To
+see the options available for this script, run:
+
+    :::bash
+    ./build-support/bin/publish_docs.sh -h
+
 To see http://www.pantsbuild.org/ site's content as it would be generated based on your local
-copy of the pants repo, enter the command
+copy of the pants repo, enter the command:
 
     :::bash
-    # This publishes the docs **locally** and opens (-o) them in your browser for review
+    # This publishes the docs in dist/docsite and opens (-o) them in your browser for review
```

Comment:
Removing the `publishes` keyword here would make it clearer that this is a sideeffect free operation (which I think was the reason for the bolding before).